modules/apps/object/object-web/src/main/resources/META-INF/resources/js/components/ModelBuilder/DefinitionNode/DefinitionNode.tsx

```diff
@@ -152,7 +152,7 @@ export function DefinitionNode({
 				/>
 
 				<NodeFooter
-					isLinkedNode={linkedDefinition}
```

Comment:
In these TSX components, I kept the same pattern mentioned earlier, so we don't have the same properties in two TSX components with different names.